maisim/maisim.Game/Screen/WarningScreen.cs

```diff
@@ -0,0 +1,108 @@
+    public class WarningScreen : osu.Framework.Screens.Screen
+    {
+        private SpriteIcon warningSprite;
+
+        private readonly osu.Framework.Screens.Screen nextScreen;
+
+        private readonly float icon_y = -100f;
+
+        public WarningScreen(osu.Framework.Screens.Screen nextScreen)
+        {
+            this.nextScreen = nextScreen;
+            ValidForResume = false;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            InternalChildren = new Drawable[]
+            {
+                new Container()
+                {
+                    Colour = Color4Extensions.FromHex("73bfe9"),
+                    RelativeSizeAxes = Axes.Both,
+                    Alpha = 0.5f,
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Scale = new Vector2(1.5f),
+                },
+                warningSprite = new SpriteIcon
+                {
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Icon = FontAwesome.Solid.ExclamationTriangle,
+                    Size = new Vector2(40),
+                    Y = icon_y,
+                    Colour = Color4.Yellow
+                },
+                new SpriteText()
+                {
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Text = "Warning :",
+                    Font = new FontUsage(size: 30),
+                    Y = -40f,
+                    Colour = Color4Extensions.FromHex("ffff7d")
+                },
+                new SpriteText()
+                {
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Text = "This project is under a heavy development process.",
+                    Font = new FontUsage(size: 30),
+                },
+                new SpriteText()
+                {
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Text = "If something goes wrong, please make an issue on the repository.",
+                    Font = new FontUsage(size: 30),
+                    Y = 40f
+                },
+            };
+        }
+
+        protected override void LoadComplete()
+        {
+            if (nextScreen != null)
+                LoadComponentAsync(nextScreen);
+        }
+
+        public override void OnEntering(ScreenTransitionEvent e)
+        {
+            base.OnEntering(e);
+
+            warningSprite.RotateTo(-10)
+                .MoveToY(icon_y - 30, 250, Easing.OutQuint)
+                .RotateTo(360, 750, Easing.OutQuint)
+                .Then()
+                .MoveToY(icon_y, 250, Easing.OutQuint);
+
+            warningSprite.FlashColour(Color4Extensions.FromHex("ffffed"), 500, Easing.OutQuint).Loop();
+
+            this.FadeInFromZero(500)
+                .Then(2500)
```

Comment:
2.5 seconds may seem a bit insufficient unless you're able to read at lightspeed. 5 seconds may be a bit better.

Reply:
The animation took like almost second, seem reasonable.